.changeset/breezy-ads-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": patch
+---
+
+fix(OnyxToast): do not cut off box shadow
```

Comment:
clarification: isn't changesets only about changes of usage, or whenever it affects the project's design aswell?